lib/gui/app.js

```diff
@@ -94,13 +94,13 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
   const currentVersion = packageJSON.version;
   const currentReleaseType = release.getReleaseType(currentVersion);
   const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates({
-    ignoreSleepUpdateCheck: currentReleaseType !== release.RELEASE_TYPE.PRODUCTION
```

Comment:
Yay, nice to see the removal of the confusing `ignoreSleepUpdateCheck` parameter :+1: